CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,217 @@
+Quansight Labs Co-Directors.
+
+In the event of an egregious incident, please report the matter formally—as
+described below—in a timely manner to the entire COC committee (`coc@quansight.com`),
+to any individual member of the COC committee, or to one of the Quansight
+Labs Co-Directors (listed above).
+
+**All reports will be kept confidential.**
+
+Incident reports will be reviewed and investigated by the COC committee. Possible
+outcomes of the process are detailed below.
+
+### a. How to Submit an Incident Report
+
+In the context of this Code of Conduct, a "report" means a written notification
+(as described below) of any Code of Conduct incident sent to any member of the COC
+committee or a Quansight Labs Co-Director.
+
+Quansight is committed to promptly addressing any incident reports in a timely manner.
+
+### b. Conflicts of Interest
+
+In the event of a conflict of interest with any COC committee member, the relevant member will recuse themselves from handling the report. Involved parties are never part of the review process.
+
+If you are concerned about submitting a report that will reach any specific individual, please contact one of the COC committee members or Quansight Labs Co-Directors directly.
+
+### c. What to Include in a Report
+
+Our ability to address any Code of Conduct incident in a timely and effective manner is critically dependent on the amount & the precision of all information provided. Please include as much of the following as possible in an incident report:
+
++ Your contact information (including your preferred medium of contact).
+    + If you wish to remain anonymous, your personal contact details will not be shared beyond
+      the person receiving the initial report.
++ Approximate time & location of the incident (be as specific as possible).
++ Identifying information of all individuals actively involved in the COC incident.
+    + This may include names, nicknames/pseudonyms, usernames/screen-names, physical
+  descriptions, etc.
++ Witnesses (if any)
+    + Provide names and contact details if available
++ Description of the circumstances/context surrounding the incident (e.g., at a conference, in a private meeting, during an online chat, etc.)
++ Description of the behavior
+    + If reporting harassing language, be specific about the words used.
+    + Provide a detailed account of what happened.
+    + Provide any available supporting records (e.g. email, screenshots, links to public GitHub
+    issues, mailing list archives, or public chat logs, etc.)
++ State whether you believe this incident is ongoing and/or part of an ongoing pattern of
+  behavior.
++ Any additional information that may be helpful.
+
+### d. Procedures after Filing a Report
+
+**Reviewing the Report**
+
+The Quansight COC committee will acknowledge receipt of the incident report within one or two business days. They will make all efforts to review the incident report within a week to determine:
```

Comment:
"within one or two business days" --> "within two business days" (it's a max, so no need to be fuzzy)